cmd/yunikorn-history-server/commands/root.go

```diff
@@ -77,7 +77,7 @@ func Run(ctx context.Context, cfg *config.Config) error {
 
 	healthService := health.New(info.Version, health.NewYunikornComponent(client), health.NewPostgresComponent(pool))
 
-	ws := webservice.NewWebService(&cfg.YHSConfig, mainRepository, eventRepository, healthService)
+	ws := webservice.NewWebService(&cfg.YHSConfig, &cfg.CORSConfig, mainRepository, eventRepository, healthService)
```

Comment:
Since cors is associated with server config, I think it might be better to have it in the `YHSConfig` struct